.github/workflows/canary-integration-test.yml

```diff
@@ -167,8 +167,7 @@ jobs:
           timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --rbd-data-pool-name replicapool --rgw-endpoint $rgw_endpoint; do sleep 1 && echo 'waiting for the rgw endpoint to be validated'; done"
           # pass the invalid rgw-endpoint of different ceph cluster
           if output=$(timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --rbd-data-pool-name replicapool --rgw-endpoint 10.108.96.128:80; do sleep 1 && echo 'waiting for the rgw endpoint to be validated'; done"); then
-            echo "unexpectedly succeeded after passing the wrong rgw-endpoint: $output"
-            exit 1
+            echo "succeeded script with stderr error after passing the wrong rgw-endpoint: $output"
```

Comment:
What does it mean to succeed if there was an error? Perhaps we should say that the script completed with errors.

Reply:
If they fix the problem, they will be able to re-run the script, right? There is nothing blocking them from running it again?

Reply:
Yes, nothing will block them, from re-running.

It is just that they still use the output